packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -149,16 +153,30 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
         resultHintString = FaceMatchStateStringMap[faceMatchState];
       }
 
+      // If the face is outside the oval set the aria-label to a string about centering face in oval
+      let a11yHintString = resultHintString;
```

Comment:
We can just continue mutating `resultHintString` here

Reply:
We have separate variables here because there are some cases where we want the a11y text to read something different than what is on screen. So in some cases on screen it will say `move closer` but for the a11y text it will read something different like `face is outside oval, center your face`

Reply:
Ahhhh, missed that `resultHintString` was still being used here